hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -16,7 +15,9 @@ class HrHolidaysPublic(models.Model):
     _order = "year"
 
     display_name = fields.Char("Name", compute="_compute_display_name", store=True)
-    year = fields.Integer("Calendar Year", required=True, default=date.today().year)
+    year = fields.Integer(
+        "Calendar Year", required=True, default=fields.Date.context_today
```

Comment:
`fields.Date.context_today == default=date.today().year`?

Reply:
`fields.Date.context_today` will return a `datetime.date` type. Not an `integer` type
![image](https://github.com/user-attachments/assets/a11c7a78-19ec-4131-b315-ff6d91325658)